From randy.dunlap@oracle.com  Tue Feb 24 21:53:34 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 19 Feb 2009 09:47:32 -0800
Subject: Staging: phison: depends on PCI
Cc: gregkh@suse.de, evan_ko@phison.com
Message-ID: <20090219094732.bacee6e1.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

phison uses PCI interfaces, so it should depend on PCI.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Evan Ko <evan_ko@phison.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/phison/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/phison/Kconfig
+++ b/drivers/staging/phison/Kconfig
@@ -1,5 +1,5 @@
 config IDE_PHISON
 	tristate "PCIE ATA PS5000 IDE support"
-	depends on ATA && ATA_SFF
+	depends on PCI && ATA && ATA_SFF
 	---help---
 	  This is an experimental driver for PS5000 IDE driver.
